src/ddscxx/include/org/eclipse/cyclonedds/core/policy/PolicyDelegate.hpp

```diff
@@ -480,6 +480,31 @@ class OMG_DDS_API OwnershipDelegate
 
 //==============================================================================
 
```

Comment:
Please also encapsulate this whole body with the conditional OMG_DDS_EXTENSIBLE_AND_DYNAMIC_TOPIC_TYPE_SUPPORT macro.

Reply:
Will do.